pkg/agent/agent_windows.go

```diff
@@ -204,6 +315,82 @@ func (i *Initializer) prepareOVSBridgeOnHNSNetwork() error {
 	return nil
 }
 
+func (i *Initializer) prepareOVSConfigForVM() error {
+	klog.InfoS("Performing OVS configuration", "hostIfName", i.nodeConfig.UplinkNetConfig.Name)
+	hostIfName := i.nodeConfig.UplinkNetConfig.Name
+	uplinkIfName := util.GenUplinkInterfaceName(hostIfName)
+	ovsPorts, err := i.ovsBridgeClient.GetPortList()
+	if err != nil {
+		klog.Errorf("Failed to list OVS ports: %v", err)
+		return err
+	} else {
+		for index := range ovsPorts {
+			port := &ovsPorts[index]
+			if port.Name == hostIfName {
+				klog.InfoS("Uplink and host configuration exists in OVS")
+				return nil
+			}
+		}
+	}
+
+	success := false
+	uplinkExternalIDs := map[string]interface{}{
+		interfacestore.AntreaInterfaceTypeKey: interfacestore.AntreaUplink,
+	}
+	// Create uplink port on OVS.
+	uplinkUUID, err := i.ovsBridgeClient.CreatePort(uplinkIfName, uplinkIfName, uplinkExternalIDs)
+	if err != nil {
+		klog.ErrorS(err, "Failed to create uplink port on OVS", "uplink", uplinkIfName)
+		return err
+	}
+
+	defer func() {
+		if !success {
+			klog.InfoS("Exiting!!  delete port on OVS", "uplinkUUID", uplinkUUID)
+			i.ovsBridgeClient.DeletePort(uplinkUUID)
+		}
+	}()
+
+	// Query the uplink port to check if its created
+	uplinkOFPort, err := i.ovsBridgeClient.GetOFPort(uplinkIfName, false)
+	if err != nil {
+		klog.ErrorS(err, "Failed to get uplink ofport", "uplink", uplinkIfName)
```

Comment:
Ditto

Reply:
I have removed  it for the VM case. 
But still there are other places where we are also logging for container case.